app/add_some_abilities/add_protection.py

```diff
@@ -0,0 +1,7 @@
+def protection(knight: dict) -> None:
+    knight["add_some_abilities"] = armour_sum(knight["armour"])

```

Comment:
The function `armour_sum` is used before its definition. Consider defining `armour_sum` before using it in the `protection` function to improve readability and avoid potential issues.